ClosedXML/Excel/AutoFilters/XLAutoFilter.cs

```diff
@@ -116,62 +116,59 @@ public XLAutoFilter Sort(Int32 columnToSortBy, XLSortOrder sortOrder, Boolean ma
             SortColumn = columnToSortBy;
 
             // Recalculate shown / hidden rows
-            if (Enabled)
```

Comment:
Why did you remove this? It seems like a good check to have?

Reply:
At the top of the method, there is already 
`if (!Enabled) throw new InvalidOperationException("Filter has not been enabled.");`

Reply:
ah ok